task-header.html

```diff
@@ -23,14 +23,16 @@
 			<div class="maintitle">
 				<h1>Danielle Catterill</h1>
 				<h2>Tea Drinker, Dog Lover & Full Stack Developer in Training</h2>
-			<div>	
+			</div>	
 		</header>
 		
-		<div><h3 class="abouttitle">About Me<h3></div>
-		
-		<div class="aboutme">
-			<img class=>
-		</div>
+		<section class="aboutme">
+				<div><img src="smiley-2055681_1920.jpg" img class="aboutmepic"></div>	
```

Comment:
for readability possibly?
```suggestion
				<div>
				           <img src="smiley-2055681_1920.jpg" img class="aboutmepic">
				 </div>	
```

Reply:
Yeah :)